core/iwasm/libraries/wasi-nn/test/CMakeLists.txt

```diff
@@ -162,12 +162,12 @@ add_executable (iwasm ${WAMR_ROOT_DIR}/product-mini/platforms/${WAMR_BUILD_PLATF
 
 install (TARGETS iwasm DESTINATION bin)
 
-target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} ${TENSORFLOW_LIB} -lm -ldl -lpthread)
+target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} ${UVWASI_LIBS} ${TENSORFLOW_LIB} -lm -ldl -lpthread)
```

Comment:
Can we `set (UV_A_LIBS "${UV_A_LIBS} ${UVWASI_LIBS}")` in lib_uvwasi.cmake so that we don't need to change these cmake files?

Reply:
Done